app/src/main/java/com/infomaniak/drive/data/sync/UploadMigration.kt

```diff
@@ -102,10 +102,19 @@ class UploadMigration : RealmMigration {
                 .addField(UploadFile::uploadHost.name, String::class.java)
             oldVersionTemp++
         }
+        //endregion
+
+        //region Migrate to version 7: Add path to MediaFolder
+        if (oldVersionTemp == 6L) {
+            schema.get(MediaFolder::class.java.simpleName)!!
+                .addField(MediaFolder::path.name, String::class.java, FieldAttribute.REQUIRED)
```

Comment:
Now we stop using `::path.name`because it can lead to crash if user jump two migration during which one renamed the field